test/CMakeLists.txt

```diff
@@ -7,6 +7,15 @@ endif()
 
 # We use the check unit testing framework for our C unit tests
 include(ExternalProject)
+if(APPLE)
```

Comment:
I think you should flip this around to be `if(NOT WIN32)`.  That way the patching path is still being tested the linux automated testing.